tests/functional/pod_and_daemons/test_mgr_pods.py

```diff
@@ -0,0 +1,136 @@
+import logging
+
+from ocs_ci.framework.testlib import (
+    tier1,
+    skipif_ocs_version,
+    skipif_ocp_version,
+    BaseTest,
+    post_upgrade,
+    post_ocs_upgrade,
+)
+from ocs_ci.ocs import exceptions
+from ocs_ci.ocs.resources import pod
+from ocs_ci.framework.pytest_customization.marks import purple_squad, brown_squad
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@skipif_ocs_version("<4.15")
+@skipif_ocp_version("<4.15")
```

Comment:
Do we need to skip OCP version too?, i.e. ODF is supported with n-1, n & n+1 OCP versions so considering this do take a call to update this.